packages/hardhat-toolbox-viem/package.json

```
@@ -68,12 +68,12 @@
     "solidity-coverage": "^0.8.1",
     "ts-node": "^10.8.0",
     "typescript": "~5.0.4",
-    "viem": "^1.15.1"
+    "viem": "^2.7.6"
   },
   "peerDependencies": {
     "@nomicfoundation/hardhat-network-helpers": "^1.0.0",
     "@nomicfoundation/hardhat-verify": "^2.0.0",
```

I know it's not related to hardhat-viem, but as we are already changing the file, we may go ahead and fix these.

```suggestion
    "@nomicfoundation/hardhat-network-helpers": "workspace:^1.0.0",
    "@nomicfoundation/hardhat-verify": "workspace:^2.0.0",
```

---

I didn't take them across as they weren't in `ethereumjs/main` and I was wanting to keep these changes as focused as possible. When we backport our release changes into `main` we definitely should add these in.